fetch-uj-records/fetch-uj-records_test.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/redhat-appstudio/segment-bridge.git/containerfixture"
+	"github.com/redhat-appstudio/segment-bridge.git/splunk"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	scriptPath   = "../scripts/fetch-uj-records.sh"
+	filePathPass = "./fetch-uj-recordsPass"
+)
+
+func compareOutputs(output []byte, filePath string) bool {
+	/*Recieves the script's output and compares it
+	with a file containing the expected output*/
+	fileOutput, err := os.ReadFile(filePath)
+	if err != nil {
+		fmt.Printf("error: %s\n", err.Error())
+		return false
+	}
+	return bytes.Equal(fileOutput, output)
+}
+
+func runAndValidateScript(t *testing.T, filePath, scriptPath string) bool {
+	/*Returns true if a given scripts runs successfully
+	and it's output is equal to a given file's content*/
+	cmd := exec.Command(scriptPath)
+	output, err := cmd.Output()
+	if err != nil {
+		fmt.Printf("error: %s\n", err.Error())
+		return false
+	}
+	return compareOutputs(output, filePath)
+}
+
+func TestFetchUjRecords(t *testing.T) {
+	/* Testing fetch-uj-records.sh script*/
+	require.NoError(t, os.Setenv("SPLUNK_APP_NAME", "-"), "Failed to set SPLUNK_API_URL")
+	require.NoError(t, os.Setenv("SPLUNK_INDEX", "test_index"), "Failed to set SPLUNK_INDEX")
+	require.NoError(t, os.Setenv("QUERY_EARLIEST_TIME", "2023-01-01T10:46:34.955712833Z"), "Failed to set QUERY_EARLIEST_TIME")
+	splunk.WithSplunkContainer(t, func(deployment containerfixture.FixtureInfo) {
+		splunkAppApiURL := splunk.GetSplunkAppAPIEndpoint("localhost", deployment.ApiPort, "nobody", "-")
+		require.NoError(t, os.Setenv("SPLUNK_API_URL", "localhost:"+deployment.ApiPort), "Failed to set SPLUNK_API_URL")
+		netrcPath, err := containerfixture.GetNetrcPath()
+		if err != nil {
+			fmt.Printf("error: %s\n", err.Error())
+			return
```

Comment:
why not assert here \ fail the test in that case? we won't be able to connect to Splunk without a netrc.